src/solve2.F

```diff
@@ -1733,22 +1733,14 @@ subroutine solve2(nstep,rbufsz,num_soil_layers,                  &
         !          conservation of total dry-air mass
         !FIXME
         !print *,'solve2: before dumk{1,2} loop: axisymm: ',axisymm
-#ifdef _B4B06R
-        !$acc update host(rho,ruh,rvh,rmh,pi0,pp3d,xf)
-#else
         !$acc parallel reduction(+:tmp1,tmp2)
```

Comment:
I would suggest that we do not inclose an if statement within a parallel region.  It would be easier to read if we moved the parallel region inside the if block.  Such a change would potentially eliminate the need for axisymm variable to be on the GPU.

Reply:
I totally agree with this comment. But moving `acc parallel` into an `if` statement could be a NBFB change if the module variable is not set properly on the device. That is why I do not make changes in this PR.

Reply:
Ok, let's move the placement of the 'acc parallel' directive on a future PR.